akka-actor/src/main/scala/akka/actor/AbstractActor.scala

```diff
@@ -5,17 +5,74 @@
 package akka.actor
 
 import akka.annotation.ApiMayChange
+import akka.japi.pf.ReceiveBuilder
+import akka.japi.pf.UnitPFBuilder
+import scala.runtime.BoxedUnit
 
 /**
  * Java API: compatible with lambda expressions
- *
- * This is an EXPERIMENTAL feature and is subject to change until it has received more real world testing.
  */
 object AbstractActor {
+
+  /**
+   * Defines which messages the Actor can handle, along with the implementation of
+   * how the messages should be processed. You can build such behavior with the
+   * [[akka.japi.pf.receivebuilder]] but it can be implemented in other ways than
+   * using the `ReceiveBuilder` since it in the end is just a wrapper around a
+   * Scala `PartialFunction`.
```

Comment:
Consider adding: `In Java, you can implement PartialFunction by extending AbstractPartialFunction.`